prisma/schema.prisma

```diff
@@ -10,25 +10,33 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model Team {
+  id    String @id @default(uuid())
+  users User[]
+
```

Comment:
nl

Reply:
hm?

Reply:
there is a newline that doesn't seem like should be there, doesn't matter though

Reply:
OHHH, next person that updates the sdk can change it, I can't change it here, it does nothing